src/util/cmdlineargs.h

```diff
@@ -67,16 +68,20 @@ class CmdlineArgs {
 
   private:
     CmdlineArgs() :
-        m_startInFullscreen(false), //Initialize vars
+        m_startInFullscreen(false), // Initialize vars
         m_midiDebug(false),
         m_developer(false),
         m_safeMode(false),
+#ifdef __WINDOWS__
+        m_settingsPath(QDesktopServices::storageLocation(QDesktopServices::DataLocation).append("/")) {
```

Comment:
What do you mean with "working on Mac"? Did the preference path change ? If there was no change, I will again use QStandardPath on Mac and treat Linux as a special case.